MediaBrowser.LocalMetadata/Parsers/BaseItemXmlParser.cs

```diff
@@ -580,7 +580,12 @@ private void FetchFromTaglinesNode(XmlReader reader, T item)
                     switch (reader.Name)
                     {
                         case "Tagline":
-                            item.Tagline = reader.ReadNormalizedString();
+                            var val = reader.ReadNormalizedString();
+                            if (!string.IsNullOrEmpty(val))
```

Comment:
This might be functionally different in that we won't set from a completely blank tagline, but that's now consistent with all the other properties.